### ISSUE TITLE
Any way to stream specific window name?

### ISSUE THREAD

**Issue by websecnl** [NeuralNine/vidstream#20]
Any way to stream specific window name?

and is there a way to simply make the remote stream control the mouse movement and clicks?

**Comment by leestarb** [comment]
No. It would be too hard to complain with the DWM. Actually the program just screenshots the screen (it is handled by pyautogui)
